sentry-spring-boot-starter/src/test/kotlin/io/sentry/spring/boot/it/SentrySpringIntegrationTest.kt

```diff
@@ -84,8 +84,8 @@ class SentrySpringIntegrationTest {
                 assertThat(event.request).isNotNull()
                 assertThat(event.request!!.url).isEqualTo("http://localhost:$port/hello")
                 assertThat(event.user).isNotNull()
-                assertThat(event.user.username).isEqualTo("user")
-                assertThat(event.user.ipAddress).isEqualTo("169.128.0.1")
+                assertThat(event.user!!.username).isEqualTo("user")
+                assertThat(event.user!!.ipAddress).isEqualTo("169.128.0.1")
```

Comment:
maybe use `assertNotNull` with the lambda so we avoid `!!` for all user asserts on this PR?